src/types.h

```diff
@@ -178,6 +178,10 @@ constexpr bool is_loss(Value value) {
 
 constexpr bool is_decisive(Value value) { return is_win(value) || is_loss(value); }
 
+constexpr bool hasMateDistance(Value absValue) {
```

Comment:
Wouldn't it be better to take `value` as parameter and compute `absValue` in the function itself?

Reply:
Yes, but it will require an additional include in types.h

Reply:
I think the additional include is fine.